python_bindings/correctness/bit_generator.cpp

```diff
@@ -0,0 +1,21 @@
+#include "Halide.h"
+
+using namespace Halide;
+
+class BitGenerator : public Halide::Generator<BitGenerator> {
+public:
+    Input<bool> bit_constant{"constant_uint1"};
+    Input<Buffer<bool>> bit_input{"input_uint1", 1};
```

Comment:
Nit: standard convention is to list all Input<Buffer> before all Input<scalar>

Reply:
Not sure I completely follow. Do you mean list all buffer Input before all constant input? So should the `Buffer<bool>` go first?

Reply:
Yes.

Reply:
Done.